core/src/main/java/com/klaytn/caver/tx/account/AccountKeyRoleBased.java

```diff
@@ -108,4 +108,13 @@ public boolean equals(Object o) {
         return Arrays.equals(toRlp(), that.toRlp());
     }
 
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        result.append("AccountKeyRoleBased : \n ");
+        result.append("RoleTransaction : \n " + getRoleTransaction().toString() + "\n ");
```

Comment:
If AccountKey is `AccountKeyFail`, `AccountKeyLegacy` or `AccountKeyNil`, `toString()` will return memory address. These objects' `toString()` do not defined.

Reply:
Okay I'll add it at this PR.